src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/casenotes/CaseNotesReconciliationService.kt

```diff
@@ -201,7 +203,7 @@ class CaseNotesReconciliationService(
     val sortedAmendments = amendments.map { a ->
       CommonAmendmentFields(
         text = a.text,
-        occurrenceDateTime = a.createdDateTime,
+        occurrenceDateTime = LocalDateTime.parse(a.createdDateTime).format(amendmentFormatter),
```

Comment:
this is really naff. The API generation generates this field as a string rather than a date time, so we need to parse it first